.env.sample

```diff
@@ -1,5 +1,4 @@
 BUGZILLA_ADMIN_KEY=<generate and copy from BZ>
-INVITE_LAMBDA_URL=<run the "invite" project on a separate port and place the url here>
```

Comment:
Don't remove it from here, as it is a reference for local development. For local development, we would want to keep this variable so it overrides the default resolving behavior